Sources/SpeziAccount/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,2090 @@
+            "value" : "oder"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "or"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "o"
+          }
+        }
+      }
+    },
+    "Package Dependencies" : {
+
+    },
+    "Password Text" : {
+
+    },
+    "Preview Service" : {
+
+    },
+    "REMOVE_DEFAULT_ERROR" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Löschen fehlgeschlagen!"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Could not remove account!"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "¡No se pudo eliminar la cuenta!"
+          }
+        }
+      }
+    },
+    "REPEAT_PASSWORD" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Bestätigen"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Verify"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Verificar contraseña"
+          }
+        }
+      }
+    },
+    "REPEAT_PASSWORD_PROMPT" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Passwort wiederholen"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "re-enter password"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Escriba la contraseña otra vez"
+          }
+        }
+      }
+    },
+    "Settings" : {
+
+    },
+    "SIGN_IN_AND_SECURITY" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "de" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Anmeldung & Sicherheit"
+          }
+        },
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Sign-In & Security"
+          }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Inicio de Sesión y Seguridad"
+          }
+        }
+      }
+    },
+    "Some Hint" : {
+
+    },
+    "The internal account identifier is meant to be generated!" : {
+
+    },
+    "The internal account identifier is not meant to be user facing!" : {
+
+    },
```

Comment:
Seems like either missing translation or again missing use of the `Text(verbatim:)` initializer